data/json/monsters/mechsuits.json

```diff
@@ -0,0 +1,139 @@
+[
+  {
+    "id": "mon_mech_recon",
+    "type": "MONSTER",
+    "name": "X-03: 'Spectre' recon mech",
+    "description": "The Boeing-Daewoo RMES ( Recon Mechanical Exoskeleton Suit ), a recent acquisition by the US military, it was designed to be used in a scout-recon-sniper role, due to its mobility and integrated laser sniper rifle and suite of optics for target designation and battlefield awareness.  It was not deployed before the Cataclsym hit, though there were a few prototypes in the field.  You may be able to hack it to accept you as its pilot.  Like all mech-suits it can act as a UPS from its large battery.",
```

Comment:
```suggestion
    "description": "The Boeing-Daewoo RMES (Recon Mechanical Exoskeleton Suit), a recent acquisition by the US military, it was designed to be used in a scout-recon-sniper role, due to its mobility and integrated laser sniper rifle and suite of optics for target designation and battlefield awareness.  It was not deployed before the Cataclysm hit, though there were a few prototypes in the field.  You may be able to hack it to accept you as its pilot.  Like all mech-suits it can act as a UPS from its large battery.",
```